src/lib/constants.ts

```diff
@@ -27,7 +27,7 @@
  */
 
 export const NAME = 'style9';
-export const SERIALIZE_COMPRESSION_FLAG = '#';
+export const SERIALIZE_COMPRESSION_FLAG = '___';
```

Comment:
The `#` comes from `vanilla-extract`'s virtual file loader.

So if we are going to change this anyway, what about making it even more distinguishable?

```suggestion
export const SERIALIZE_COMPRESSION_FLAG = '__SECRET_SERIALIZE_COMPRESSION_FLAG_DO_NOT_USE_OR_YOU_WILL_BE_FIRED__';
```